McJon/northern-pet-swag
# Landing page 2022

### Goal
Build a Summer Promotion Landing Page

### Testing Instructions
Visit https://vnz4eh7u0bfrfbg0-61011132598.shopifypreview.com/pages/summer-promotion and confirm that it matches the wireframe.

### Designs
[Wireframe](https://www.northern.co/media/7236/download?attachment)

### Reviewers
Justin
